broadway/read-model-mongodb/0.2/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "container": {
+        "mongodb_host": "localhost",
+        "mongodb_port": "27107",
```

Comment:
These parameters build up a DSN. We should consider using the DSN here directly. That would reduce the number of parameters.